schedule/systemd/suse_patches-systemd_testsuite.yaml

```diff
@@ -19,12 +46,6 @@ schedule:
     - systemd_testsuite/test_13_nspawn_smoke
     - systemd_testsuite/test_14_machine_id
     - systemd_testsuite/test_15_dropin
-    - systemd_testsuite/test_16_extend_timeout
-    - systemd_testsuite/test_17_udev_wants
-    - systemd_testsuite/test_18_failureaction
-    - systemd_testsuite/test_19_delegate
-    - systemd_testsuite/test_20_mainpidgames
-    - systemd_testsuite/test_21_sysusers
     - systemd_testsuite/test_22_tmpfiles
-    - systemd_testsuite/test_23_type_exec
+    - {{systemd_testsuite_15SP2ORLATER}}
```

Comment:
Looks like we need some quotation marks here: `- '{{systemd_testsuite_15SP2ORLATER}}'`
Added them in another pull request: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10141
Failing tests:
- [Tumbleweed](https://openqa.opensuse.org/tests/1248884)
- [Leap 15.2](https://openqa.opensuse.org/tests/1248455)

Reply:
The quotes were added already in another PR, so rebasing to master should help.

Reply:
Thanks for the info!